scripts/macos/RenameMacUserNameAndHomeDirectory.sh

```diff
@@ -1,61 +1,137 @@
-#!/bin/bash
-
-#
-# Script to rename the user name of a user on OS X
-#
-# The script updates the users record name and home directory
-# name from an old name to a new one.
-#
-# NOTE: MUST BE RUN AS ROOT!
-#
+#!/bin/bash -eu
+
+: '
+ Script to rename the username of a user account on OS X
+
+ The script updates the users record name, RealName (displayName), and home directory
+
+ If the user recieving the name change is signed in they will be signed out. 
+
+ Example usage: 
+
+ sudo sh RenameMacUserNameAndHomeDirectory.sh cat dog
+
+ Above example would rename account cat to dog
+
+ NOTE: SCRIPT MUST BE RUN AS ROOT!
+'
+
 abort() {
```

Comment:
Since you're no longer using this function you can remove it, or make use of it. Your choice.